Lib/dis.py

```diff
@@ -308,8 +311,33 @@ def _get_name_info(name_index, name_list):
     return argval, argrepr
 
 
+def parse_varint(iterator):
+    b = next(iterator)
+    val = b & 63
```

Comment:
Just curious, how does this work? I guess the constant numbers are about 64-bit? 

would love to get your explanation, thanks. @markshannon